apps/etterlatte-vedtaksvurdering/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/VedtakForEksterntService.kt

```diff
@@ -0,0 +1,34 @@
+package no.nav.etterlatte.no.nav.etterlatte.vedtaksvurdering
+
+import no.nav.etterlatte.libs.common.person.Folkeregisteridentifikator
+import no.nav.etterlatte.libs.common.vedtak.VedtakForEkstern
+import no.nav.etterlatte.libs.common.vedtak.VedtakForEksternUtbetaling
+import no.nav.etterlatte.libs.common.vedtak.VedtakForEksterntDto
+import no.nav.etterlatte.libs.common.vedtak.VedtakTypeForEkstern
+import no.nav.etterlatte.vedtaksvurdering.VedtakInnhold
+import no.nav.etterlatte.vedtaksvurdering.VedtaksvurderingRepository
+
+class VedtakForEksterntService(
+    private val repository: VedtaksvurderingRepository,
+) {
+    fun hentVedtak(fnr: Folkeregisteridentifikator): VedtakForEksterntDto {
+        val vedtak = repository.hentFerdigstilteVedtak(fnr)
+        return VedtakForEksterntDto(
+            vedtak =
+                vedtak.filter { it.type.vanligBehandling }.map {
+                    val vedtakInnhold = (it.innhold as VedtakInnhold.Behandling)
```

Comment:
Det er vel ingen garanti for at denne alltid blir rett hvis personen har en klage eller tilbakekreving feks?